NeoWeb/Helper.cs

```diff
@@ -41,15 +42,15 @@ public static string UploadMedia(IFormFile cover)
             var bytes = new byte[10];
             random.NextBytes(bytes);
             var newName = bytes.ToHexString() + Path.GetExtension(cover.FileName);
-            var filePath = Path.Combine(Directory.GetCurrentDirectory(), "wwwroot/upload", newName);
```

Comment:
After the asp.net 2.2 Directory. GetCurrentDirectory() fails, the official recommended using dependency injection mode for the current Directory.

Reply:
Then we can remove https://github.com/neo-project/neo.org/blob/8b26bd1226478b9fa0f6b79dd23a8b8070c31de5/NeoWeb/Helper.cs#L16